### PR TITLE
Add ESEF 2022 conformance suite

### PR DESCRIPTION
**Pull request by derekgengenbacher-wf**
#### Reason for change
ESEF 2022 has been released and we need to be testing against the ESEF 2022 conformance suite

#### Steps to Test
CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by austinmatherne-wk — DISMISSED** [review]

File: tests/integration_tests/validation/conformance_suite_configurations/esef_ixbrl_2022.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from tests.integration_tests.validation.conformance_suite_config import ConformanceSuiteConfig
+
+config = ConformanceSuiteConfig(
+    args=[
+        '--disclosureSystem', 'esef',
+        '--formula', 'run',
+        '--plugins', 'validate/ESEF_2022',
+    ],
+    expected_failure_ids=frozenset([
+        # Both of the following test cases fail because of the `tech_duplicated_facts1` formula which
```
Comment (austinmatherne-wk) [inline tests/integration_tests/validation/conformance_suite_configurations/esef_ixbrl_2022.py:11]: Refers to `both`, but the second test was removed from this list

**Review by austinmatherne-wk — APPROVED** [review]

**Review by allenbross-wf — APPROVED** [review]

**Review by aaroncameron-wk — APPROVED** [review]

File: tests/integration_tests/validation/conformance_suite_configs.py [hunk 2]
```diff
@@ -25,7 +27,9 @@
 ALL_CONFORMANCE_SUITE_CONFIGS: tuple[ConformanceSuiteConfig, ...] = (
     efm_current,
     esef_ixbrl_2021,
+    esef_ixbrl_2022,
```
Comment (aaroncameron-wk) [inline tests/integration_tests/validation/conformance_suite_configs.py:30]: Do we have an idea for how long we want to retain/run the 2021 conformance suite? For the recent EFM update, we just updated in place. Is there a reason we wouldn't want to do that here?

Comment (derekgengenbacher-wf) [reply tests/integration_tests/validation/conformance_suite_configs.py:30]: We will most likely retain the 2021 suite until ESEF 2021 and older taxonomies are all deprecated and no longer accepted by authorities.

**Pull request merged**